packages/css/src/components/tile/tile.scss

```diff
@@ -0,0 +1,15 @@
+.dso-whitebox-small { // DEPRECATED
```

Comment:
Ik geloof bijna niet dat je dit getest hebt.. Want waar is de rest van de styling op `.dso-whitebox-small` ?